soc/st/stm32/stm32u5x/Kconfig.defconfig.stm32u545xx

```diff
@@ -0,0 +1,11 @@
+# ST Microelectronics STM32U545XX MCU
+
+# Copyright (c) 2024 Opito
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32U545XX
+
+config NUM_IRQS
+	default 114
```

Comment:
It should be the highest IRQ number + 1
```suggestion
	default 126
```

Reply:
I derived this number from the datasheet. In the U575, the NVIC section mentioned 125 maskable interrupts (matching the defconfig). Can you point me where I can find this info?

Reply:
![image](https://github.com/user-attachments/assets/21b77693-c68a-463e-9f22-de9006c8f11c)
Also the 575 currently has `125` as the value for `NUM_IRQS` too. Should that be `126`?

Reply:
Strangely the only definition I've found is for arc architecture, but the same applies to all arch:
```
config NUM_IRQS
	int "Upper limit of interrupt numbers/IDs used"
	range 17 256
	help
	  Interrupts available will be 0 to NUM_IRQS-1.
```